PegSolitaire.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include <iostream>
+#include <chrono>
+#include <fstream>
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <array>
+#include <unordered_set>
+#include <tuple>
+#include <regex>

```

Comment:
Include only what you need
![image](https://github.com/user-attachments/assets/d1f5ad32-4823-41f4-a81f-3ca2ea93da8e)

Reply:
Removed #using <tuple>, others are used indirectly in .c file.